axelar-chains-config/tests/schema/index.js

```diff
@@ -25,24 +25,28 @@ export const axelarGatewaySchema = {
     id: '/info/chains.contracts.AxelarGateway',
     type: 'object',
     properties: {
-        connectionType: { type: 'string' },
+        connectionType: {
+            type: 'string',
+            enum: ['consensus', 'amplifier'],
+        },
     },
     required: ['connectionType'],
+    additionalProperties: true,
 };
 
 export const contractSchema = {
     id: '/info.chains.contracts',
     type: 'object',
     patternProperties: {
-        // PascalName e.g. 'AxelarGasService', 'AxelarGateway' etc.
-        '(^[a-z]|[A-Z])[a-z]*': {
+        // PascalName e.g. 'AxelarGasService' etc.
+        '^(?!AxelarGateway)(^[a-z]|[A-Z])[a-zA-Z]*$': {
```

Comment:
it's fine to only have AxelarGateway as the available contract. That is the only required protocol contract technically so no need to exclude matching for it?